### PR TITLE
16994 update viewport bot docs

### PR DESCRIPTION
**Pull request by holdenhinkle**
Update Cypress Viewport Updater Bot documentation about having to manually update a vcr cassette.

### CODE REVIEW

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md [hunk 1]
```diff
@@ -57,3 +57,49 @@ The `Cypress Viewport Updater Bot` GitHub App is owned by `@department-of-vetera
 Secrets for for the Google API Service Account and the GitHub App are stored in credstash. The Google Analytics settings are available in the `google_analytics_cvu` object in `config/settings.yml` and the GitHub settings are available in the `github_cvu` object in `config/settings.yml`. The settings are available in the `dev`, `staging`, `prod`, and `sandbox` environments.
 
 All sensitive Google API and GitHub settings that VCR normally records in each cassette `yml` file has been filtered out.
```
Comment (pjhill) [inline platform/testing/end-to-end/automatic-cypress-viewport-updater.md:59]:
```suggestion
All sensitive Google API and GitHub settings that VCR normally records in each cassette `yml` file have been filtered out.
```

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md [hunk 2]
```diff
@@ -57,3 +57,49 @@ The `Cypress Viewport Updater Bot` GitHub App is owned by `@department-of-vetera
...
+[2021-02-09T22:12:49.377Z] 
+[2021-02-09T22:12:49.377Z]   1) EVSS::DisabilityCompensationForm::SubmitForm0781.perform_async with a successful submission job submits successfully
+[2021-02-09T22:12:49.377Z]      Failure/Error: if (match = interaction.request.body.match(/^grant_type.+/))
+[2021-02-09T22:12:49.377Z] 
+[2021-02-09T22:12:49.377Z]      ArgumentError:
+[2021-02-09T22:12:49.377Z]        invalid byte sequence in UTF-8
+[2021-02-09T22:12:49.377Z]      # ./spec/jobs/cypress_viewport_updater/google_analytics_reports_spec.rb:20:in `match'
+```
+
+The spec `spec/jobs/cypress_viewport_updater/google_analytics_reports_spec.rb` was causing another test to file, while the spec itself was passing. A most unusual interaction was occuring between spec files.
```
Comment (pjhill) [inline platform/testing/end-to-end/automatic-cypress-viewport-updater.md:74]:
```suggestion
The spec `spec/jobs/cypress_viewport_updater/google_analytics_reports_spec.rb` was causing another test to fail, while the spec itself was passing. A most unusual interaction was occurring between spec files.
```

**Review by pjhill — CHANGES_REQUESTED** [review]
Typos

Otherwise, this LGTM. What a weird behavior!

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md (hunk 1 above)
Comment (holdenhinkle) [reply platform/testing/end-to-end/automatic-cypress-viewport-updater.md]: Thanks Peter! Fixed!

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md (hunk 2 above)
Comment (holdenhinkle) [reply platform/testing/end-to-end/automatic-cypress-viewport-updater.md]: Thanks Peter! Fixed!

**Review by billfienberg — APPROVED** [review]

**Pull request merged**